template/test-report.hbs

```diff
@@ -55,15 +55,19 @@
 
     <div class="layout-stacked__content">
       <div class="layout-container--short" style="padding-top: 35px;">
-        <div class="cards--vuln filter--patch filter--ignore">
-          {{#each vulnerabilities}}
-          {{> vuln-card showSummaryOnly=../showSummaryOnly}}
-          {{/each}}
-        </div><!-- cards -->
+        {{#if vulnerabilities}}
+          <div class="cards--vuln filter--patch filter--ignore">
+            {{#each vulnerabilities}}
+            {{> vuln-card showSummaryOnly=../showSummaryOnly}}
+            {{/each}}
+          </div><!-- cards -->
+        {{else}}
+         No known vulnerabilities detected.
```

Comment:
can we test in test? see examples in https://github.com/snyk/snyk-to-html/blob/master/test/snyk-to-html.test.ts
both when we have vulns and do not

Reply:
added, checking why 1 test is unhappy with what seems like the same diff :D